types/scaleway-functions/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for scaleway-functions 1.0
```

Comment:
This line should be `// Type definitions for non-npm package scaleway-functions 1.0`, to indicate that the original package is not on npm. Then with this change, I think you can re-enable the "npm-naming" rule on the tslint config below.

Reply:
Thanks, I've applied the suggested changes :)